### PR TITLE
[exporter/f5cloud] use generated status header

### PR DESCRIPTION
**Pull request by codeboten**
This PR updates the component to generate the status table using mdatagen.

Linked issue: #21213

### CODE REVIEW

**Review by crobert-1 — APPROVED** [review]

**Review by gramidt — APPROVED** [review]

**Review by codeboten — COMMENTED** [review]
@gramidt it looks like the link to `https://portal.cloudservices.f5.com/` is causing a failure, do you happen to know where that link should point?

**Comment by gramidt** [comment]
@codeboten - Sadly, I'm not sure. I left F5, but I will reach out to someone who would know and who should be the new code owner for this exporter.

**Review by mwear — APPROVED** [review]

**Comment by astencel-sumo** [comment]
Here's the standalone issue for the broken link: https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/22077

**Review by andrzej-stencel — APPROVED** [review]

File: exporter/f5cloudexporter/README.md [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
+| ------------- |-----------|
+| Stability     | [beta]: traces, metrics, logs   |
+| Distributions | [contrib], [observiq] |
+
+[beta]: https://github.com/open-telemetry/opentelemetry-collector#beta
+[contrib]: https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib
+[observiq]: https://github.com/observIQ/observiq-otel-collector
+<!-- end autogenerated section -->
 
 Exports data via HTTP to [F5 Cloud](https://portal.cloudservices.f5.com/).
```
Comment (codeboten) [inline exporter/f5cloudexporter/README.md:14]:
```suggestion
Exports data via HTTP to F5 Cloud.
```

**Pull request merged**